assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -1008,6 +1008,28 @@
                         <!-- End VeriCite custom options -->
 					#elseif($reviewServiceName.equals("TurnitinOC"))
 						<!-- Start TurnitinOC custom options -->
+						#if($show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT)
+							<div class="form-required">
+								<label>$tlang.getString("review.originality.reports")</label>
+							</div>
+							<div class="well" style="display:inline-block">
+								#if($show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT.contains($name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_IMMEDIATELY))
+									<div class="radio">
+										<label><input type="radio" name="$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_RADIO" value="$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_IMMEDIATELY" #if($value_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_RADIO.equals("$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_IMMEDIATELY"))checked#end/>$tlang.getString("review.originality.reports.immediately")</label>
+									</div>
+								#end
+								#if($show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT.contains($name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_DUE))
+									<div class="radio">
+										<label><input type="radio" name="$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_RADIO" value="$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_DUE" #if($value_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_RADIO.equals("$name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_DUE"))checked#end/>$tlang.getString("review.originality.reports.due")</label>
+									</div>
+								#end
+								#if($show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT.contains($name_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT_IMMEDIATELY_AND_DUE))
+									<div class="radio">
```

Comment:
Can you explain how "show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT" works? Is this a Sakai property where you can show/hide one (or all 3) options? Is there a sakai.property to set the default selection? Have you QA'ed the properties?

Reply:
In sakai properties you set which options are viewable with 
turnitin.report_gen_speed.setting=0,1,2 (which will show all three)

These options are stored as a list in reportGenOptions which are then put on line 2585
context.put("show_NEW_ASSIGNMENT_REVIEW_SERVICE_REPORT", reportGenOptions);

Note/Possible bug here - if option 0 is excluded from turnitin.report_gen_speed.setting the assignment radio will be "un checked" when creating a new assignment since option 0 is the default checked. You can override this setting but setting setting.value default to a another setting.

example
turnitin.report_gen_speed.setting=2
turnitin.report_gen_speed.setting.value=2

which shows:

<img width="289" alt="screen shot 2018-03-20 at 11 41 22 am" src="https://user-images.githubusercontent.com/26496920/37665444-a7eefb60-2c33-11e8-80bf-6af0d276c75e.png">

It would be possible to change the default behavior but would involve changing more AssignmentAction code, and would only be applicable when 0 or "immediately" would be hidden which may not be applicable.